Backend.Core/ItemsManagers/LayersManager.cs

```diff
@@ -2,33 +2,65 @@
 using Backend.Models.RoomItems;
 using System.Collections.Generic;
 using System.Linq;
+using Serilog;
 namespace Backend.Core
 {
     internal class LayersManager : IRoomItemsManager
     {
-        private readonly Dictionary<string, Layer> _layers = new Dictionary<string, Layer>();
+        //private readonly Dictionary<string, Layer> _layers = new Dictionary<string, Layer>();
+        private List<Layer> _layers = new List<Layer>();
         public int Count => _layers.Count;
-        private List<Layer> LayerList { get => _layers.Values.ToList(); }
+        private int layerNameCount = 1;
 
         public bool Add(IRoomItem item) {
             var layer = (Layer)item;
-            layer.Id = $"Layer {Count + 1}";
-            return _layers.TryAdd(layer.Id, layer);
+            layer.Id = $"Layer {layerNameCount}";
+            layerNameCount++;
+            _layers.Add(layer);
+            return true;
         }
 
-        public bool Delete(string Id) => _layers.Remove(Id);
+        public bool Delete(string Id)
+        {
+            bool result = _layers.Remove((Layer) Get(Id));
+            if (!result)
+            {
+                string layersStr = "";
+                for (int i = 0; i < _layers.Count; i++) layersStr += "\"" + _layers[i].Id + "\", ";
```

Comment:
use string interpolation $"{layseStr} text {_layers[i].Id}"

Reply:
Dont you miss brackets  { } for the loop?

Reply:
Yes